qt/widgets/instrumentview/src/InstrumentWidgetDecoder.cpp

```diff
@@ -113,6 +113,7 @@ void InstrumentWidgetDecoder::decodeRenderTab(const QMap<QString, QVariant> &map
   obj->m_lighting->setChecked(map[QString("displayLighting")].toBool());
   obj->m_GLView->setChecked(map[QString("useOpenGL")].toBool());
   obj->m_UCorrection->setChecked(map[QString("useUCorrection")].toBool());
+  obj->m_maintainAspectRatio->setChecked(map[QString("maintainAspectRatio")].toBool());
```

Comment:
Can I just check that this safely loads old projects that won't have `maintainAspectRatio` in the list of saved (encoded) options?

Reply:
I just tried this and yes an old project without the setting works OK. `toBool()` returns false if the setting is missing which I think is fine in that the instrument viewer loads in the same state it would have been with the old Mantid code ie without the aspect ratio maintained.

Reply:
Perfect. Thanks!